rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -154,15 +154,22 @@ void Player::play_messages_until_queue_empty(const PlayOptions & options)
   }
 }
 
+rclcpp::QoS Player::publisher_qos_for_topic(const std::string & topic_name)
+{
+  auto qos_it = topic_qos_profile_overrides_.find(topic_name);
+  if (qos_it != topic_qos_profile_overrides_.end()) {
+    ROSBAG2_TRANSPORT_LOG_INFO_STREAM("Overriding QoS profile for topic " << topic_name);
+    return Rosbag2QoS{topic_qos_profile_overrides_.at(topic_name)};
```

Comment:
```suggestion
    return Rosbag2QoS{qos_it->second};
```
you've already searched and found it

Reply:
Yup, I realized that when going through the PR again :sweat_smile: